app/main.py

```diff
@@ -1,2 +1,118 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Hashable, Any, Iterable
+
+INITIAL_CAPACITY = 8
```

Comment:
These constants should be defined as class attrs since it is a part of Dict implementation and will not be used outside of it